_ru/tour/basics.md

```diff
@@ -26,187 +26,306 @@ previous-page: tour-of-scala
 ## Выражения
 
 Выражения — это вычислимые утверждения.
+
+{% tabs expression %}
+{% tab 'Scala 2 и 3' for=expression %}
 ```scala mdoc
```

Comment:
I ran a linter on this `.md` files and got an edit with indents before and after ```, maybe add these new lines according to the general rules of linter md files? 
For e.x.

```
(line separation)
'''scala mdoc
1 + 1
'''
(line separation)
```

For autolint [`prettier`](https://prettier.io/) could be used :)

Reply:
Thanks for the link :) 
I'll use it